api/src/main/java/openconsensus/stats/data/MeasureType.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019, OpenConsensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package openconsensus.stats.data;
+
+/**
+ * An enum that represents all the possible value types for a {@code Measure} or a {@code
+ * Measurement}.
+ *
+ * @since 0.1.0
+ */
+public enum MeasureType {
```

Comment:
Move type in the measure.

Reply:
Done. Note this `Type` is also used in Measurement.